manager/src/main/java/org/openremote/manager/asset/AssetStorageService.java

```diff
@@ -604,17 +604,18 @@ public List<String> findNames(String... ids) {
      * @throws IllegalArgumentException if the realm or parent is illegal, or other asset constraint is violated.
      */
     public <T extends Asset<?>> T merge(T asset) throws IllegalStateException, ConstraintViolationException {
-        return merge(asset, false);
+        return merge(asset, false, false);
     }
 
     /**
      * @param overrideVersion If <code>true</code>, the merge will override the data in the database, independent of
      *                        version.
+     * @param preserveTimestamps    If true, the attributes being merged will be stored with their stored timestamp, given those timestamps are newer than the current asset's attributes.
      * @return The current stored asset state.
      * @throws IllegalArgumentException if the realm or parent is illegal, or other asset constraint is violated.
      */
-    public <T extends Asset<?>> T merge(T asset, boolean overrideVersion) throws IllegalStateException, ConstraintViolationException {
-        return merge(asset, overrideVersion, false, null);
+    public <T extends Asset<?>> T merge(T asset, boolean overrideVersion, boolean preserveTimestamps) throws IllegalStateException, ConstraintViolationException {
```

Comment:
Don't want to do this unless really necessary would rather solve this problem without a configurable option for consistency; an overload method to maintain the original method signature should have been added

Reply:
This is for-sure not going to remain like this in the future, it is part of the debugging attempts at